### DISCUSSION TITLE
Can the main keyboard shortcuts be changed from Alt to Ctrl?

### DISCUSSION THREAD

**Discussion by RedBearAK** [Ulauncher/Ulauncher#944] [General]
Is it possible to edit something and change all of the shortcuts to launch results list items from using `Alt` to `Ctrl`? The shortcuts preferences seem to only apply to shortcuts for searching.

**Comment by friday** [comment]
No. It's not possible. I think Alt was picked because Ctrl is sometimes used for other things, but that's only my guess.
